### DISCUSSION TITLE
Allow going back go the previous page

### DISCUSSION THREAD

**Discussion by TheAirBlow** [ppy/osu#27594] [Ideas] [answered]
So whenever you e.g. open a beatmap from a user's profile in osu!lazer, pressing escape would instead close the window instead of going back, which is mildly annoying as loading the profile tales ages and you have to click "load more" a bunch of times...

If there is a way already or this is too difficult to implement, let me know!

**Comment by Joehuu** [comment, marked as answer]
Already tracked in https://github.com/ppy/osu/issues/5063.
